apps/omg_watcher/test/omg_watcher/integration/in_flight_exit_test.exs

```diff
@@ -257,7 +257,82 @@ defmodule OMG.Watcher.Integration.InFlightExitTest do
            end) =~ "Invalid in-flight exit finalization"
 
     assert %{"in_flight_exits" => [_], "byzantine_events" => byzantine_events} = WatcherHelper.success?("/status.get")
-    assert [%{"event" => "invalid_piggyback"}] = Enum.filter(byzantine_events, &(&1["event"] != "piggyback_available"))
+
+    assert [%{"event" => "unchallenged_piggyback"}, %{"event" => "invalid_piggyback"}] =
+             Enum.filter(byzantine_events, &(&1["event"] != "piggyback_available"))
+  end
+
+  @tag fixtures: [:in_beam_watcher, :alice, :bob, :mix_based_child_chain, :token, :alice_deposits]
```

Comment:
we're stepping away from using fixtures. find another way to test this scenarios